shapeshift/web
# Add XDeFi wallet to SelectWallet Modal

## Description
Added XDeFi Wallet as an option in the SelectWallet Modal.

### Please describe your changes
I have added a new option in the SelectWallet modal for connecting with XDeFi Wallet. I have also created a [PR](https://github.com/shapeshift/hdwallet/pull/388) in the hdwallet repo, to support the option with its own `XDeFi Adapter`. Currently supporting the Ethereum chain.

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)
fixes #720 
## Testing
1. Pull branch locally and run `yarn` to install new deps
2. Run webapp and have xdefi wallet installed
3.  Click on XDeFi option when prompted to connect
4. Ensure that wallet is connect by asserting that the XDeFi pop up prompts the user to select their accounts



## Screenshots (if applicable)
